content/blog/2023-12-11-arrays-and-variable-arguments-in-Bril.md

```diff
@@ -0,0 +1,96 @@
++++
+title = "Variable Arguments in Bril and Arrays in the TypeScript compiler"
+[extra]
+bio = """
+ Alice is an M.Eng student studying Computer Science. She is broadly interested in compilers, systems and algorithm design.
+"""
+[[extra.authors]]
+name = "Alice Sze"
++++
+
+# What was the goal?
```

Comment:
I don't think you need to include section headings that directly mirror my bullet points from [the syllabus](https://www.cs.cornell.edu/courses/cs6120/2023fa/syllabus/#experience-report). Remember that your imaginary audience here is people who are generally interested in compilers and your work but do not particularly care that this was done as a 6120 project. Do you think there's a different way to title these sections that would more accurately reflect the contents and help such a reader more quickly navigate your writing?